### ISSUE TITLE
Not getting the results from dreambooth lora training

### ISSUE THREAD

**Issue by Sandeep-Narahari** [huggingface/diffusers#5014]
accelerate launch train_dreambooth_lora_sdxl.py --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" --instance_data_dir="/home/test/testt" --pretrained_vae_model_name_or_path="madebyollin/sdxl-vae-fp16-fix" --output_dir="/home/test/sandeep" --mixed_precision="fp16" --instance_prompt="a photo of sandeep" --resolution=1024 --train_batch_size=1 --gradient_accumulation_steps=4 --learning_rate=1e-4 --train_text_encoder --with_prior_preservation --enable_xformers_memory_efficient_attention --lr_scheduler="constant" --lr_warmup_steps=0 --max_train_steps=500 --validation_prompt="a photo sandeep" --validation_epochs=25 --seed="23" --class_data_dir="/home/test/classsandeep" --class_prompt="a photo of sandeep"

have given  6 images of mine in dataset 

Tried multiple times , but not getting the images as per the training dataset, followed [https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/README_sdxl.md](https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/README_sdxl.md)

@williamberman, @sayakpaul, @yiyixuxu
Could you please let me know where I am missing 

Thank you

**Comment by sayakpaul** [comment]
This question should go to ou Discord forum since it directly doesn't concern our codebase.

**Comment by yuxu915** [comment]
hi, @Sandeep-Narahari , I meet the same question, cannot generate results as good as that in redeme.me, do you finally find where the problems are and solve them? thanks.
